jcl/src/java.base/share/classes/openj9/internal/tools/attach/target/DiagnosticUtils.java

```diff
@@ -288,6 +293,19 @@ private static DiagnosticProperties runGC() {
 		return DiagnosticProperties.makeCommandSucceeded();
 	}
 
+	private static DiagnosticProperties getJstatClass(String diagnosticCommand) {
+		IPC.logMessage("jstat command : ", diagnosticCommand); //$NON-NLS-1$
+		StringWriter buffer = new StringWriter(100);
+		PrintWriter bufferPrinter = new PrintWriter(buffer);
+		bufferPrinter.println("Class Loaded    Class Unloaded"); //$NON-NLS-1$
+		// "Class Loaded".length = 12, "Class Unloaded".length = 14
+		bufferPrinter.printf("%12d    %14d", //$NON-NLS-1$
+				Integer.valueOf(ClassLoaderInfoBaseImpl.getLoadedClassCountImpl()),
+				Long.valueOf(ClassLoaderInfoBaseImpl.getUnloadedClassCountImpl()));
+		bufferPrinter.println();
```

Comment:
If you add `%n` to the end of the format string, this call to `println()` can be removed.

Reply:
Agreed.